File: android/app/src/main/java/org/vechain/veworld/app/minimizer/RCTMinimizer.java
package org.vechain.veworld.app.minimizer;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.ReactMethod;

public class RCTMinimizer extends ReactContextBaseJavaModule {
    RCTMinimizer(ReactApplicationContext context) {
        super(context);
    }

    @Override
    public String getName() {
        return "Minimizer";
    }

    @ReactMethod
    public void goBack() {
        android.app.Activity activity = getCurrentActivity();
        if (activity != null) {
            activity.moveTaskToBack(true);
        }
    }
}

File: android/app/src/main/java/com/veworld/minimizer/RCTMinimizer.kt
package com.veworld.minimizer

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule
import com.facebook.react.bridge.ReactMethod

class RCTMinimizer(reactContext: ReactApplicationContext) :
    ReactContextBaseJavaModule(reactContext) {
    override fun getName(): String = "Minimizer"

    @ReactMethod
    fun goBack() {
        val activity = currentActivity
        activity?.moveTaskToBack(true)
    }
}